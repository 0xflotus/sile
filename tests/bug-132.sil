\begin[papersize=a5]{document}
\script[src=packages/verbatim]
\font[family=DejaVu Sans Mono,size=9pt]
\define[command=verbatim:font]{\font[family=DejaVu Sans Mono,size=9pt]}
\begin{verbatim}
#          #
##();:,i[]##
##MMWWQQGG##
\end{verbatim}
\font[family=Gentium Book Basic,size=9pt]
\begin{verbatim}
#          #
##();:,i[]##
##MMWWQQGG##
\end{verbatim}
\end{document}
