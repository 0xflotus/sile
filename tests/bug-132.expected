<tests/bug-132.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	20.97638
My 	36.22374
Set font 	DejaVu Sans Mono;9;200;normal;normal;;LTR
T	6	(#)
Mx 	80.57941
T	6	(#)
Mx 	20.97638
My 	46.06163
T	6	(#)
Mx 	26.39484
T	6	(#)
Mx 	31.81329
T	11	(()
Mx 	37.23175
T	12	())
Mx 	42.65021
T	30	(;)
Mx 	48.06866
T	29	(:)
Mx 	53.48712
T	15	(,)
Mx 	58.90558
T	76	(i)
Mx 	64.32403
T	62	([)
Mx 	69.74249
T	64	(])
Mx 	75.16095
T	6	(#)
Mx 	80.57941
T	6	(#)
Mx 	20.97638
My 	57.00255
T	6	(#)
Mx 	26.39484
T	6	(#)
Mx 	31.81329
T	48 48 58 58 52 52 42 42	(MMWWQQGG)
Mx 	75.16095
T	6	(#)
Mx 	80.57941
T	6	(#)
Mx 	20.97638
My 	80.64903
T	6	(#)
Mx 	80.57941
T	6	(#)
Mx 	20.97638
My 	90.48692
T	6	(#)
Mx 	26.39484
T	6	(#)
Mx 	31.81329
T	11	(()
Mx 	37.23175
T	12	())
Mx 	42.65021
T	30	(;)
Mx 	48.06866
T	29	(:)
Mx 	53.48712
T	15	(,)
Mx 	58.90558
T	76	(i)
Mx 	64.32403
T	62	([)
Mx 	69.74249
T	64	(])
Mx 	75.16095
T	6	(#)
Mx 	80.57941
T	6	(#)
Mx 	20.97638
My 	101.42784
T	6	(#)
Mx 	26.39484
T	6	(#)
Mx 	31.81329
T	48 48 58 58 52 52 42 42	(MMWWQQGG)
Mx 	75.16095
T	6	(#)
Mx 	80.57941
T	6	(#)
[1] Mx 	207.41759
My 	553.73265
Set font 	Gentium;10;200;normal;normal;;LTR
T	17	(1)
End page
Finish

